src/Components/RegAssistCodeBlock/RegAssistCodeBlock.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  CodeBlock,
+  CodeBlockAction,
+  CodeBlockCode,
+  ClipboardCopyButton,
+} from '@patternfly/react-core';
+
+const RegAssistCodeBlock = ({ code }) => {
```

Comment:
nit-pick:
```suggestion
const RegAssistCodeBlock = ({ code = [] }) => {
```

or tweak proptypes's `code` to be required